JSDemos/Demos/TreeView/HierarchicalDataStructure/description.md

```diff
@@ -1 +1,5 @@
-The TreeView component displays a&nbsp;hierarchical collection of&nbsp;items using a&nbsp;traditional tree structure. TreeView nodes can be&nbsp;expanded and collapsed to&nbsp;display sub-items.
\ No newline at end of file
+To bind the TreeView to a hierarchical array, assign this array to the [items[]](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/) property. If you need to use a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object and its diverse API, specify the [dataSource](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataSource) property instead.
+
+No matter which property is used, each hierarchical node should include a `text`, a unique `id`, optional developer-defined fields, and an `items` array if a node nests others. If the **items[]** TreeView property is specified, fields from its API can also be included.
```

Comment:
Let's make a list of mandatory fields (`id`, `text`, `items`) and give their descriptions, then say that these names can be different and give links to the Expr properties. Finally, say that there are more node properties (cite `expanded` as an example) and give a link to the items section in the API reference.